browser/mapping.go

```diff
@@ -130,25 +130,50 @@ func mapResponse(ctx context.Context, vu k6modules.VU, r api.Response) mapping {
 	return maps
 }
 
+// mapJSHandle to the JS module.
+func mapJSHandle(ctx context.Context, vu k6modules.VU, eh api.JSHandle) mapping {
```

Comment:
Minor: rename from `eh` to `jsh`?

Reply:
Resolved [here](https://github.com/grafana/xk6-browser/blob/d50f4ba1a7934371c122116fdcf2c8653618a08d/browser/mapping.go#L134).